src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -77,6 +77,73 @@ public Task Get()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/databases/topology", "GET", "/admin/databases/topology?&name={databaseName:string}")]
+        public Task GetTopology()
```

Comment:
* Need to change the route
* Need to overwrite the url of the current node with the provided url from the user.